### DISCUSSION TITLE
Reason not to have a restart policy for Docker Compose services?

### DISCUSSION THREAD

**Discussion by zdeneksvarc** [postalserver/postal#2136] [Development discussions]
Hello guys,

the topic of auto-start has been discussed in several threads, but nowhere have I found a satisfactory answer.

I may have missed something, but a look at the Postal v2.1.1 `/usr/bin/postal` file shows that `postal start` command is a simple wrapper for the `docker compose up` command without any magic. If so, is there any reason not to have a restart policy in `docker-compose.yml` for Postal v2?

So I added a `restart: always` for each service in the 
`/opt/postal/install/docker-compose.yml` and `/opt/postal/install/templates/docker-compose.yml` and `run docker compose -p postal up -d` by hand to set up restart policy.

After several forced system restarts, the Postal server does not seem to have any problems and auto-start up fine. Am I missing something, please?

**Comment by kluvi** [comment]
Hi.. just found this old pull request: https://github.com/postalserver/install/pull/3 (I have not idea, why its not already merged)

**Reply by willpower232** [reply]
I think we need @adamcooke to have some time to review and merge it

**Reply by bonest** [reply]
Looks like the PR is still not merged?

I am a new user of postal server. Did my first setup a few weeks ago, and have just learned that I have to start postal manually on reboot. 

I hope this PR gets merged into the next update.

**Comment by jshapiro2** [comment]
Did this ever happen?
